routes/spec.js

```diff
@@ -1128,27 +1133,26 @@ Currently the API has a rate limit of **60 requests/minute** and **50,000 reques
             },
           },
         },
-        route: () => '/skyblock/bazaar/:id',
-        func: (req, res, cb) => {
-          const itemId = req.params.id;
-          redis.get('skyblock_bazaar', (err, resp) => {
-            const ids = JSON.parse(resp) || [];
-            if (itemId && !itemId.includes(',') && !ids.includes(itemId)) {
-              return res.status(400).json({ error: 'Invalid itemId' });
+        route: () => '/skyblock/bazaar/:id?',
```

Comment:
Looks like a question mark sneaked in to the route here

Reply:
This is intentional, since the bazaar now returns all items if none are specified.